src/templates/pt/2019/error.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base/2019/error.html" %}
+
+{% block unknown_error %}Erro desconhecido{% endblock %}
```

Comment:
We added these in #1166 
```suggestion
{% block date_published %}2019-11-04T12:00:00.000Z{% endblock %}
{% block date_modified %}2020-08-06T00:00:00.000Z{% endblock %}

{% block unknown_error %}Erro desconhecido{% endblock %}
```